### ISSUE TITLE
Autoupdate client does not honor WebAppInternals.setBundledJsCssPrefix

### ISSUE THREAD

**Issue by egoldblum** [meteor/meteor#7149]
Currently running 1.2.1 on Galaxy.

We set `WebAppInternals.setBundledJsCssPrefix` within a `Meteor.startup`.  In the general case this works fine.

Observed behavior:
After deploying new code, connected browser clients reload automatically, but requests for the new js/css land on our origin not the CDN.

Expected behavior:
Autoupdate'd css `link href` (and javscript `script src`) should continue to respect the set `WebAppInternals.setBundledJsCssPrefix`

https://github.com/meteor/meteor/blob/devel/packages/autoupdate/autoupdate_client.js#L131

**Comment by laosb** [comment]
Yeah, should be and did reproduce on a self-hosted app.

**Comment by tmeasday** [comment]
A simple way to reproduce this locally: 

```bash
# in one terminal
meteor create testapp
cd testapp
echo "WebAppInternals.setBundledJsCssPrefix('http://localhost:4000')" >> server/main.js
meteor

# in another
git clone https://github.com/tmeasday/simple-proxy
cd simple-proxy
npm install
node index.js
```

**Comment by tmeasday** [comment]
@egoldblum I am only seeing the behavior you are referring to for CSS files, not JS files. Are you sure JS files don't hit the CDN?

**Comment by egoldblum** [comment]
@tmeasday you are correct, only rewritten css links cause requests to our origin, not js.

**Comment by abernix** [comment]
This is confirmed fixed in Meteor 1.3.4.2!  Thanks for reporting this problem, @egoldblum.
